main.tf

```diff
@@ -91,4 +91,12 @@ resource "aws_route_table_association" "public_subnet_asso" {
     count = length(var.public_subnet_cidrs)
     subnet_id = element(aws_subnet.public_subnets[*].id, count.index)
     route_table_id = aws_route_table.second_rt.id
+}
+
+resource "aws_instance" "fedora" {
+    ami = "ami-06b7f026f48cd3c6b"
+    instance_type = "t2.micro"
```

Comment:
You need to add a key pair, or you won't be able to log in to your new server.

Reply:
Good catch! I'll update it.

Reply:
Done!

Reply:
Done!